### PR TITLE
Rename Doc/README.txt to Doc/README.rst

### PR DESCRIPTION
**Pull request by ghost**
I have simply updated the document and its references to README.rst (thanks to whoever formatted it).

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately our records indicate you have not signed the CLA. For legal reasons we need you to sign this before we can look at your contribution. Please follow these steps to rectify the issue:

1. Sign the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/)
2. **Wait at least a day** and then check "Your Details" on [bugs.python.org](https://bugs.python.org/) to see if your account has been marked as having signed the CLA (the delay is due to a person having to manually check your signed CLA)
3. Reply here saying you have completed the above steps


Thanks again to your contribution and we look forward to looking at it!

**Review by methane — APPROVED** [review]

File: Doc/make.bat [hunk 1]
```diff
@@ -74,7 +74,7 @@ echo.   Provided by this script:
 echo.      clean, check, serve, htmlview
 echo.
 echo.All arguments past the first one are passed through to sphinx-build as
-echo.filenames to build or are ignored.  See README.txt in this directory or
+echo.filenames to build or are ignored.  See README.rst in this directory or
```
Comment (methane) [inline Doc/make.bat:77]: `Doc/README.txt` is still alive.

**Review by methane — CHANGES_REQUESTED** [review]

File: Doc/make.bat (hunk 1 above)
Comment (zware) [reply Doc/make.bat:77]: This PR renames Doc/README.txt to Doc/README.rst.

Comment (methane) [reply Doc/make.bat:77]: Ah, I'm sorry.  I misundersttod.

**Review by methane — APPROVED** [review]

**Comment by methane** [comment]
I think we don't require CLA sign for pull requests like this.

**Comment by orsenthil** [comment]
@methane  , any patch / contribution requires CLA sign.

**Comment by ghost** [comment]
I signed it an hour ago.

**Comment by methane** [comment]
> **Wait at least a day** and then check "Your Details" on bugs.python.org to see if your account has been marked as having signed the CLA (the delay is due to a person having to manually check your signed CLA)

**Review by vstinner — CHANGES_REQUESTED** [review]
I suggest to take the opportunity of this change to not only rename README.txt to README.rst but also make some reST syntax enhancements like using reST syntax for links.

**Comment by ghost** [comment]
@methane MY CLA still has not been marked, and I currently have this:

image

It's been two days since I submitted it, any update?

**Comment by vstinner** [comment]
Why was this change tagged as "backport to 3.6"? Was it made in the 3.6 branch? If yes, please redo it in the master branch.

**Comment by ghost** [comment]
No, it was made in the master branch @haypo

**Comment by ghost** [comment]
@haypo Can you give me an example as to how you would like the link formatted? I'm more used to writing in markdown than I am in reST

**Comment by vstinner** [comment]
> @haypo Can you give me an example as to how you would like the link formatted? I'm more used to writing in markdown than I am in reST

Look at README.rst in the root directory.

File: Doc/README.rst [hunk 2]
```diff
@@ -109,10 +109,10 @@ Contributing
 ============
 
 Bugs in the content should be reported to the Python bug tracker at
-https://bugs.python.org.
+`bugs.python.org <https://bugs.python.org>`_.
 
 Bugs in the toolset should be reported in the Sphinx bug tracker at
-https://www.bitbucket.org/birkenfeld/sphinx/issues/.
+`bitbucket.org/birkenfeld/sphinx/issues/ <https://www.bitbucket.org/birkenfeld/sphinx/issues/>`_.
```
Comment (vstinner) [inline Doc/README.rst]: I suggest "Bugs in the toolset should be reported in the [Sphinx bug tracker]."

File: Doc/README.rst [hunk 3]
```diff
@@ -109,10 +109,10 @@ Contributing
 ============
 
 Bugs in the content should be reported to the Python bug tracker at
```
Comment (vstinner) [inline Doc/README.rst]: "Bugs in the content should be reported to the [Python bug tracker]."

File: Doc/README.rst [hunk 4]
```diff
@@ -3,19 +3,19 @@ Python Documentation README
...
 
 
 Building the docs
 =================
 
-You need to have Sphinx <http://sphinx-doc.org/> installed; it is the toolset
+You need to have `Sphinx <http://sphinx-doc.org/>`_ installed; it is the toolset
 used to build the docs.  It is not included in this tree, but maintained
-separately and available from PyPI <https://pypi.python.org/pypi/Sphinx>.
+separately and available from `PyPI <https://pypi.python.org/pypi/Sphinx>`_.
```
Comment (vstinner) [inline Doc/README.rst]: and [available from PyPI].

File: Doc/README.rst [hunk 5]
```diff
@@ -3,19 +3,19 @@ Python Documentation README
 
 This directory contains the reStructuredText (reST) sources to the Python
 documentation.  You don't need to build them yourself, prebuilt versions are
-available at <https://docs.python.org/dev/download.html>.
+available at `docs.python.org/dev/download.html <https://docs.python.org/dev/download.html>`_.
```
Comment (vstinner) [inline Doc/README.rst]: ..., [prebuilt versions are available].

**Review by vstinner — CHANGES_REQUESTED** [review]
I suggested different text where [...] is the link.

**Comment by ghost** [comment]
The reason the CLA bot is not showing me as signed is probably because I used the online editor for the original commit (which commits under the noreply@github address). My email address is `narayan<>roger<>@<>gmail.com` (remove the <>)

**Comment by ghost** [comment]
@haypo The changes you requested were made

**Comment by ghost** [comment]
It is still not showing up on my account! I have filed a new one for each day it has not been added.

**Comment by berkerpeksag** [comment]
Please note that the PSF is based in the United States and they can only process CLA requests in workdays.

**Comment by ghost** [comment]
I am moving this PR soon now that my CLA is confirmed and due to overlaps with the master branch.

**Comment by ghost** [comment]
Moved to #104 and my CLA is now processed

**Pull request closed without merging**